ios/Classes/core/PHAsset+PM_COMMON.m

```diff
@@ -94,6 +94,26 @@ - (NSString *)title {
         return @"";
     }
 }
+
+- (NSString *)originalFileName {
```

Comment:
Now since we support subtypes, could we also support obtaining the title from the resource that matched the subtype? Also, the `titleAsync` might have an argument like `bool withSubtype = false`.

Reply:
The type is PHAssetResourceType, not PHAssetMediaType

Reply:
https://developer.apple.com/documentation/photokit/phassetresourcetype?language=objc

Reply:
If we need to get the originalFileName based on a different resource type, may need to return a Map<ResourceType, String>

Reply:
> The type is PHAssetResourceType, not PHAssetMediaType

Indeed. But when we fetch videos for live photos, we first check the `PHAssetMediaType` (subtype), then fetch resources accordingly.

https://github.com/fluttercandies/flutter_photo_manager/blob/a8c193ed320bed5e833fcb7a93998e1652f9fec8/ios/Classes/core/PMManager.m#L422-L423

https://github.com/fluttercandies/flutter_photo_manager/blob/a8c193ed320bed5e833fcb7a93998e1652f9fec8/ios/Classes/core/PMManager.m#L445-L446

Reply:
like this
<img width="556" alt="image" src="https://user-images.githubusercontent.com/14145407/159666911-5ddb7c7f-acd9-48f4-9df3-19db400c3884.png">